docs/community/com-ingest/com-ingest-source-kafka-static.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Kafka Static Ingest Reference
+layout: default
+parent: Import data
+grand_parent: Community
+---
+
+## Kafka Static Ingester
+
+The Kafka Static ingester reads JSON messages from Kafka and writes the data to FeatureBase. It uses a static schema file which is also written in JSON. 
+
```

Comment:
Need a Before you begin/prereqisites heading here.

It's simplest to just hyperlink to what needs to be in place.